MATLAB/Utilities/addCTnoise.m

```diff
@@ -7,7 +7,7 @@
 %   'Poisson' : changes the average photon count for Poisson noise. Default
 %               value is 1e5
 %   'Gaussian': changes the mean and standard deviation of the electronic
-%               Gaussian noise. Default value is [0 10]
+%               Gaussian noise. Default value is [0 0.5]

```

Comment:
According to the existing code....
https://github.com/CERN/TIGRE/blob/7e761f505e3c156a816d77fcc8570333a8ed528d/MATLAB/Utilities/addCTnoise.m#L79-L82